packages/@aws-cdk/aws-lambda/lib/log-retention-provider/index.ts

```diff
@@ -108,4 +113,19 @@ export async function handler(event: AWSLambda.CloudFormationCustomResourceEvent
       }
     });
   }
+
+  function parseRetryOptions(rawOptions: any) {
+    const retryOptions: { maxRetries?: number, retryOptions?: { base?: number } } = {};
```

Comment:
Might be nice to take advantage of type checking (instead of using `any`) and define a type like

```ts
interface SdkRetryOptions {
  readonly maxRetries?: number;
  retryOptions?: AWS.RetryDelayOptions;
}
```

`parseRetryOptions` will return this type and the options parameter in APIs like `createLogGroupSafe` and `setRetentionPolicy()` can be set to this type.